### PR TITLE
Clarify STI acronym [ci-skip]

### PR DESCRIPTION
**Pull request by cbliard**
I'm not familiar with this acronym and I had to look up it. It does not mean Sexually Transmitted Infection as google suggests.

### CODE REVIEW

**Review by p8 — APPROVED** [review]
Thanks!

**Review by ceritium — APPROVED** [review]

**Review by jonathanhefner — COMMENTED** [review]
Good idea! :laughing:

File: guides/source/upgrading_ruby_on_rails.md [hunk 1]
```diff
@@ -534,7 +534,7 @@ While applications should stick to that interface, the actual Zeitwerk loader ob
 Rails.autoloaders.main
 ```
 
-That may be handy if you need to preload STIs or configure a custom inflector, for example.
+That may be handy if you need to preload STIs (Single Table Inheritance) or configure a custom inflector, for example.
```
[the remark this answers is not in the document]
Comment (cbliard) [reply guides/source/upgrading_ruby_on_rails.md]: yes, it's better. Thanks.

File: guides/source/upgrading_ruby_on_rails.md [hunk 2]
```diff
@@ -554,7 +554,7 @@ All is good!
 
 All known use cases of `require_dependency` have been eliminated, you should grep the project and delete them.
 
-If your application has STIs, please check their section in the guide [Autoloading and Reloading Constants (Zeitwerk Mode)](autoloading_and_reloading_constants.html#single-table-inheritance).
+If your application uses Single Table Inheritance (STI), please check their section in the guide [Autoloading and Reloading Constants (Zeitwerk Mode)](autoloading_and_reloading_constants.html#single-table-inheritance).
```
[the remark this answers is not in the document]
Comment (cbliard) [reply guides/source/upgrading_ruby_on_rails.md]: I missed that. Thanks.

**Comment by jonathanhefner** [comment]
@cbliard That looks good!  Would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?  Then I'll give this a merge!

**Comment by cbliard** [comment]
@jonathanhefner sure, I just did it. Thank you for linking to the documentation about squashing. That was helpful.

**Comment by jonathanhefner** [comment]
Thank you, @cbliard! :smiley:

**Pull request merged**